sea-query-binder/src/sqlx_mysql.rs

```diff
@@ -73,6 +73,8 @@ impl sqlx::IntoArguments<'_, sqlx::mysql::MySql> for SqlxValues {
                 Value::ChronoDateTimeWithTimeZone(t) => {
                     args.add(Value::ChronoDateTimeWithTimeZone(t).chrono_as_naive_utc_in_string());
                 }
+                #[cfg(feature = "postgres-interval")]
+                Value::Interval(_) => {}
```

Comment:
Is there a better of doing this other than splitting the query builder or throwing an error?